yargs.js

```diff
@@ -922,7 +922,7 @@ function Yargs (processArgs, cwd, parentRequire) {
 
   self.terminalWidth = function () {
     argsert([], 0)
-    return process.stdout.columns
+    return process.stdout.columns || null
```

Comment:
this is breaking tests because some environments report the width as `0` (OSX on Travis specifically).

What if we replace this with:

`return typeof process.stdout.columns !== 'undefined' ? process.stdout.columns : null`